app/views/SettingsView/index.js

```diff
@@ -112,6 +112,11 @@ class SettingsView extends React.Component {
 		Share.share({ message: isAndroid ? PLAY_MARKET_LINK : APP_STORE_LINK });
 	}
 
+	saveToClipboard = (content) => {
+		Clipboard.setString(content);
+		Alert.alert('Copied to clipboard');
```

Comment:
Can you show a toast instead?
Here's an example: https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/develop/app/containers/MessageActions.js#L259

Reply:
Actually even I didn't like the Alert box but I could not recall the word 'toast' even though I was thinking of toast box itself. Then I thought of 'modal' and 'alert'. So I mentioned that I used Alert so that a better alternative could be suggested  . Thanks for the review